src/main/java/Activities.java

```diff
@@ -0,0 +1,34 @@
+public class Activities {
+    private String Activity;
+    private int status = 0; // 0 represent undone: 1 represents done
+
+    public Activities(String activity) {
+        Activity = activity;
+    }
+    public static Activities newActivity(String activity){
+        return new Activities(activity);
+    }
+
+    public void doIt() {
+        status = 1;
+    }
+
+    public String getActivity(){
+        return Activity;
+    }
+    public int isDone(){
+        return status;
+    }
+
+    @Override
+    public String toString() {
+        String listChecker;
```

Comment:
The name listChecker is a little misleading may come across more like a method that does the action of checking rather than a variable. Perhaps you can consider something like itemStatus or activityStatus or something to that effect? The same applies for your Todo, Deadline and Event classes.